src/Sylius/Bundle/ApiBundle/Resources/config/services/serializers.xml

```diff
@@ -156,13 +156,13 @@
             <tag name="serializer.normalizer" priority="64" />
         </service>
 
-        <service id="Sylius\Bundle\ApiBundle\Serializer\TranslatableLocaleKeyDenormalizer">
+        <service id="sylius_api.denormalizer.locale_key.translatable" class="Sylius\Bundle\ApiBundle\Serializer\Denormalizer\TranslatableLocaleKeyDenormalizer">
             <tag name="serializer.normalizer" priority="96" />
         </service>
 
-<!--        <service id="Sylius\Bundle\ApiBundle\Serializer\ProductVariantChannelPricingsChannelCodeKeyDenormalizer">-->
-<!--            <tag name="serializer.normalizer" priority="96" />-->
-<!--        </service>-->
+        <service id="sylius_api.denormalizer.channel_pricing_channel_code_key.product_variant" class="Sylius\Bundle\ApiBundle\Serializer\Denormalizer\ProductVariantChannelPricingsChannelCodeKeyDenormalizer">
```

Comment:
```suggestion
        <service id="sylius_api.denormalizer.product_variant_channel_pricing_channel_code_key" class="Sylius\Bundle\ApiBundle\Serializer\Denormalizer\ProductVariantChannelPricingsChannelCodeKeyDenormalizer">
```
🤔

Reply:
I saw in other places that we are also pointing out the resource at the end in that kind of serializers, just thinking if we should clarify it, becouse some of them are now like that and others like you suggest

Reply:
But these are different cases, for example: `sylius_api.denormalizer.numeric_to_string.tax_rate` - there is one implementation (`NumericToStringDenormalizer`) used by multiple resources depending on a passed argument. Here, in your case, there is a specific serializer for the product variant.

However, it is not a blocker for this PR, it could be refactored later 🖖🏻